src/minizinc/CLI/driver.py

```diff
@@ -149,6 +146,18 @@ async def create_process(
             solver (Union[str, Path, None]): Solver configuration string
                 guaranteed by the user to be valid until the process has ended.
         """
+        kwargs = {
+            "stdin": None,
+            "stdout": PIPE,
+            "stderr": PIPE,
+        }
+        if sys.platform == "win32":
+            # On Windows, MiniZinc needs to be in its own console (but also hidden)
```

Comment:
Do you have any reference as to why this is the case? It would be great if we could include a description of what exactly this workaround is doing, why it is necessary, and if there are any potential changes that would be able to resolve the difference between windows and other OSes at a later stage.

Reply:
I've added a comment with a more thorough explanation.